protocols/autonat/src/protocol.rs

```diff
@@ -18,12 +18,12 @@
 // FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 // DEALINGS IN THE SOFTWARE.
 
-use crate::structs_proto;
+use crate::protos::structs as structs_proto;
```

Comment:
I am not a fan of renaming modules on import. It would be cool if we could just access all structs under a `proto` namespace.

To achieve, would it make sense if we:

1. Rename the `protos` module to `generated`
2. Create a `proto` module in `lib.rs` as `mod proto {}`
3. Re-export all types that we need from there to flatten out the module hierarchy.